client/src/Components/Overview/Gallery.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+
+
+const Gallery = () => {
+  const [productURL, setProductUrl] = useState('');
+  const [styles, setStyles] = useState([]);
+  const [currentStyle, setCurrentStyle] = useState([]);
+
+  const getProduct = () => {
+    axios.get('/products/40348/styles')
+      .then((res) => {
+        setProductUrl(res.data.results[0].photos[0].url);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  useEffect(() => {
+    axios.get('/products/40348/styles')
```

Comment:
consider moving this into its own function

Reply:
Can we talk about this?